packages/fxa-support-panel/.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": [
+    "react-app",
```

Comment:
Support panel doesn't have React either. Side note, we hope to [convert it to EJS](https://github.com/mozilla/fxa/issues/12998) for consistency 😎